src/vector/app.tsx

```diff
@@ -169,13 +169,6 @@ async function verifyServerConfig(): Promise<IConfigOptions> {
         const isUrl = config["default_is_url"];
 
         const incompatibleOptions = [wkConfig, serverName, hsUrl].filter((i) => !!i);
-        if (incompatibleOptions.length > 1) {
-            // noinspection ExceptionCaughtLocallyJS
-            throw new UserFriendlyError(
-                "Invalid configuration: can only specify one of default_server_config, default_server_name, " +
-                    "or default_hs_url.",
-            );
-        }
```

Comment:
I think we still need to handle `hsUrl` being specified alongside the other options